src/main/resources/assets/ae2_mega_things/lang/ja_jp.json

```
@@ -0,0 +1,34 @@
+{
+    "chat.ae2_mega_things.OnlyEmptyDISKsCanBeDisassembled": "分解ができるのは空のDISKのみです。",
```

ここも[AE2の翻訳](https://github.com/AppliedEnergistics/Applied-Energistics-2/blob/c1c95077e2a763a511a3a6a79d3118d538a0f39d/src/main/resources/assets/ae2/lang/ja_jp.json#L242)に寄せたほうがいいですね。

```suggestion
    "chat.ae2_mega_things.OnlyEmptyDISKsCanBeDisassembled": "空のDISKドライブのみが分解できます。",
```